grpcx/server.go

```diff
@@ -51,7 +51,7 @@ func WithRequestResponseLogs(l logx.Logger) grpc.UnaryServerInterceptor {
 			})
 		}
 
-		l.Info("gRPC Message", fields...)
+		l.Debug("gRPC Message", fields...)
```

Comment:
I'll keep these as INFO, I believe that DEBUG level is too low, and it doesn't appear on DD.